netbox/extras/lookups.py

```diff
@@ -14,4 +15,18 @@ def as_sql(self, qn, connection):
         return 'CAST(LENGTH(%s) AS BOOLEAN) != %s' % (lhs, rhs), params
 
 
+class NetContainsOrEquals(Lookup):
+    """
+    This lookup has the same functionality as the one from the ipam app except lhs is cast to inet
+    """
+    lookup_name = 'net_contains_or_equals'
```

Comment:
Have you considered using a different name for this lookup? Maybe something like `contains_ipaddress`? I don't know if we _should_ differentiate it from the existing `net_contains_or_equals` lookup; just throwing it out there.

Reply:
I think the current name makes sense as it's the same lookup just working with a different datatype. I'll leave it for now, if you feel like it would be better with another name, feel free to change it when merging 👍